### PR TITLE
[qpid-proton] update and don't build tests

### PR DESCRIPTION
**Pull request by autoantwort**

### CODE REVIEW

**Comment by Thomas1664** [comment]
I think the patch is still needed. When I updated OpenSSL to 3.0.2, qpid-proton just failed on Linux

File: ports/qpid-proton/fix-openssl3.patch [hunk 1]
```diff
@@ -1,22 +0,0 @@
-diff --git a/CMakeLists.txt b/CMakeLists.txt
```
Comment (JonLiu1993) [inline ports/qpid-proton/fix-openssl3.patch:1]: @autoantwort ,Thanks for your contribution, could you please clarify why remove this patch?

Comment (autoantwort) [reply ports/qpid-proton/fix-openssl3.patch:1]: Because I now pass `-DENABLE_WARNING_ERROR=OFF`, so the number of warnings is not important anymore.

**Review by JonLiu1993 — APPROVED** [review]

**Review by strega-nil-ms — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**